### ISSUE TITLE
no minimap2 index was found

### ISSUE THREAD

**Issue by johnstonmj** [StevenWingett/FastQ-Screen#76]
I was trying to use fastq_screen with --aligner "minimap2".

I encountered the error:
`Skipping DATABASE 'genome_xyz' since no minimap2 index was found at '/home/references/genome_xyz/genome_xyz.fa'`

I see that this is looking for a minimap2 index ending with a fasta file extension.

```
                } elsif ( $aligner eq 'minimap2' ) {    #minimap2
                    unless ( -e "$db_path.fa" or -e "$db_path.fa.gz" ) {
                        warn "Skipping DATABASE '$db_name' since no minimap2 index was found at '$db_path'\n";
						$database_skipped_flag = 1;
                        next;
                    }
```

I think this may be improved by replacing "fa" with "mmi", as mmi is the expected index file extension.

```
                } elsif ( $aligner eq 'minimap2' ) {    #minimap2
                    unless ( -e "$db_path.mmi" ) {
                        warn "Skipping DATABASE '$db_name' since no minimap2 index was found at '$db_path'\n";
						$database_skipped_flag = 1;
                        next;
                    }
```

As a temporary workaround, I can symlink the mmi index to match the tested ".fa" file extension:

`ln -s "genome_xyz.mmi" "genome_xyz.fa.fa"`

This is confusing as the "fa" file extension should indicate a fasta file, and this results in a double ".fa.fa" extension if $db_path already ends with ".fa"

Hope this helps!
Thanks for a great tool!
Excited for the inclusion of minimap2!

**Comment by sandratra-rab** [comment]
I agree, my current workaround is to keep the original fasta/fa.gz files in the same directory as the index/.mmi files for now.
Otherwise it works well !
